src/components/PostInfo/PostInfo.scss

```diff
@@ -1 +1,16 @@
-//
+.PostInfo {
+  margin: 10px auto;
```

Comment:
Avoid using inline styles like 'width: 90%;'. Consider creating a reusable CSS class or using CSS variables to handle common styling.